docker/kernels/geonotebook/kernel.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
Pointing out an inconsistency here:
In the master we've been able to use this `kernel.json` on both local and yarn case
where is the `geodocker-jupyter-geonotebook` repo still has two kernels
https://github.com/geodocker/geodocker-jupyter-geopyspark/tree/master/kernels